src/app/src/main/java/com/example/bookmark/MyBookDetailsActivity.java

```diff
@@ -24,6 +29,8 @@
  */
 public class MyBookDetailsActivity extends BackButtonActivity implements MenuOptions {
     private static final int EDIT_REQUEST_CODE = 101;
+    private static final int GET_ISBN_TO_GIVE_BOOK = 1;
+    private static final int GET_ISBN_TO_RECEIVE_BOOK = 1;
```

Comment:
Should GET_ISBN_TO_GIVE_BOOK and GET_ISBN_TO_RECEIVE_BOOK both equal one? Is there a special reason these numbers were chosen?

Reply:
nice catch lol im dumb

Reply:
apart from both numbers mistakenly being set to 1, the numbers were chosen arbitrarily. Let me know if there is some pattern we should be using to assign request codes